packages/cli/src/build.ts

```diff
@@ -198,7 +206,39 @@ function buildSingleFile(
         log('[Build]', 'output transpiled css');
         tryRun(() => fs.writeFileSync(cssAssetOutPath, cssCode), `Write File Error: ${outPath}`);
     }
-    // .d.ts?
+    // .d.ts
+    if (dts) {
+        const dtsContent = createModuleSource(res, 'dts', false);
+
+        log('[Build]', 'output .d.ts');
+        tryRun(
+            () => fs.writeFileSync(outSrcPath + '.d.ts', dtsContent),
+            `Write File Error: ${outPath}`
+        );
+
+        // .d.ts.map
+        // if not explicitly defined, assumed true with "--dts" parent scope
+        if (dtsSourceMap !== false) {
+            const dtsMappingContent = createModuleSource(
```

Comment:
please don't use `createModuleSource` it wasn't a good stab for generating dts.map. I don't like the undefined

Reply:
Will extract.